pkgs/desktops/elementary/services/elementary-settings-daemon/default.nix

```diff
@@ -0,0 +1,139 @@
+{ fetchurl, substituteAll, stdenv, meson, ninja, pkgconfig, gnome3, perl, gettext, glib, libnotify, lcms2, libXtst
+, libxkbfile, libpulseaudio, alsaLib, libcanberra-gtk3, upower, colord, libgweather, polkit
+, geoclue2, librsvg, xf86_input_wacom, udev, libgudev, libwacom, libxslt, libxml2, networkmanager
+, docbook_xsl, wrapGAppsHook, python3, ibus, xkeyboard_config, tzdata, nss, elementary, accountsservice }:
+
+stdenv.mkDerivation rec {
+  pname = "elementary-settings-daemon";
+  version = "3.30.1.2";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-settings-daemon/${stdenv.lib.versions.majorMinor version}/gnome-settings-daemon-${version}.tar.xz";
+    sha256 = "079dh609rvpwfyzg4m898q8km9g7x04hg18rwwb1izj1dr7zdp2w";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
+      inherit tzdata;
+    })
+    (fetchurl {
+      url = https://git.launchpad.net/~ubuntu-desktop/ubuntu/+source/gnome-settings-daemon/plain/debian/patches/45_suppress-printer-may-not-be-connected-notification.patch?id=ubuntu/3.30.1.2-1ubuntu3;
```

Comment:
I actually don't like how I'm fetching all these patches because it's a pain to update. Though it isn't frequent.

Perhaps I could fetch this repo as another src and just apply all the patches during the `patchPhase` manually.